### PR TITLE
Meetups -- expand options w and w/o speakers

### PR DESCRIPTION
**Pull request by mjang**
Replaces PR #123 

I've consolidated all new speaker ideas (mine and Margaret's) into the existing Speakers section

### CODE REVIEW

File: docs/organizer-guide/meetups/starting.rst [hunk 1]
```diff
@@ -126,15 +126,38 @@ Speakers:
 Try different formats. Alternatives:
 
 * Panels,
```
Comment (plaindocs) [inline docs/organizer-guide/meetups/starting.rst:128]: to be picky we should probably lose this comma

**Review by plaindocs — APPROVED** [review]
:+1:

**Review by ericholscher — APPROVED** [review]
Looks good. 👍 

Feel free to merge.

**Comment by mjang** [comment]
I plan to merge tomorrow, to give @meker12 a chance to re-review.

**Comment by mjang** [comment]
Merging -- and this will close issue #72

**Pull request merged**